pinot-core/src/main/java/com/linkedin/pinot/core/query/pruner/PartitionSegmentPruner.java

```diff
@@ -32,6 +34,7 @@
  * Implementation of {@link SegmentPruner} that uses partition information to perform pruning.
  */
 public class PartitionSegmentPruner extends AbstractSegmentPruner {
+  public static final String IN_CLAUSE_DELIMITER = "\t\t";
```

Comment:
Isn't the delimiter defined somewhere in the RangePredicate class? If not it belongs there.